test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -91,6 +91,11 @@ class AccountConnectionWizard:
         "type": "QLineEdit",
         "visible": 1,
     }
+    VIRTUAL_File_RADIO_BUTTON = names.syncModeGroupBox_useVfsRadioButton_QRadioButton
```

Comment:
```suggestion
    VIRTUAL_FILE_RADIO_BUTTON = names.syncModeGroupBox_useVfsRadioButton_QRadioButton
```
Just to follow the naming convention.